__tests__/news/news.query.test.js

```diff
@@ -52,14 +52,9 @@ describe('News queries tests', () => {
               }
             }
           `,
-          context: {
-            headers: {
-              token,
-            },
-          },
+          context: { headers: { token } },
```

Comment:
Why do you need this line if the `context` is set up when you call `setupApp`?

Reply:
Fixed